src/frontend/apps/e2e/__tests__/app-desk/team.spec.ts

```diff
@@ -33,7 +33,15 @@ test.describe('Team', () => {
 
     await expect(page.getByText(`1 member`)).toBeVisible();
 
-    await expect(page.getByText(`Created at 06/02/2024`)).toBeVisible();
-    await expect(page.getByText(`Last update at 07/02/2024`)).toBeVisible();
+    const today = new Date(Date.now());
+    const todayFormated = today.toLocaleDateString('en', {
+      month: '2-digit',
+      day: '2-digit',
+      year: 'numeric',
+    });
```

Comment:
It feels weird to test the code by duplicating it, but I guess there is no better way to do it. Isn't it possible to freeze the date? Might be hard as it's involving front and back container.